controllers/pwa.js

```diff
@@ -44,7 +45,8 @@ router.get('/', (req, res, next) => {
         hasPreviousPage: pageNumber > 1,
         nextPageNumber: pageNumber + 1,
         previousPageNumber: pageNumber - 1,
-        currentPageNumber: pageNumber
+        currentPageNumber: pageNumber,
+        GOOGLE_ANALYTICS: GOOGLE_ANALYTICS
```

Comment:
I think if you add `GOOGLE_ANALYTICS` to [`app.locals`](https://github.com/GoogleChrome/gulliver/blob/cc77640bfd577464c583eb8871cc885ca5ed8d11/app.js#L35) it will be automatically available to all templates. (Then it wouldn't need to be added manually to every template.)

Might also be better to to give `GOOGLE_ANALYTICS` a camel case name, to match other variables.